lib/components/ConfirmationDialog.jsx

```diff
@@ -101,16 +116,20 @@ ConfirmationDialog.propTypes = {
     okButtonText: PropTypes.string,
     cancelButtonText: PropTypes.string,
     isInProgress: PropTypes.bool,
+    isOkButtonEnabled: PropTypes.bool,
+    buttonCssClass: PropTypes.string,
 };
 
 ConfirmationDialog.defaultProps = {
     title: 'Confirm',
     text: null,
     children: null,
     isInProgress: false,
+    isOkButtonEnabled: true,
     onCancel: null,
     okButtonText: 'OK',
     cancelButtonText: 'Cancel',
+    buttonCssClass: 'core-btn',
```

Comment:
Is this ever changed from the default? I fear this might be feature bloat. Perhaps can be deleted?

Reply:
Yes, this can be deleted. Will do.